src/css/vars.css

```diff
@@ -31,16 +31,16 @@
   /* nav */
   --nav-background: var(--ds-background-body);
   --nav-border-color: var(--ds-divider);
-  --nav-line-height: 1.35;
+  --nav-line-height: 1.5;
   --nav-heading-font-color: var(--ds-text-tertiary);
   --nav-muted-color: var(--ds-text-tertiary);
   --nav-panel-divider-color: var(--ds-divider);
   --nav-secondary-background: var(--ds-background-body);
   /* toolbar */
   --toolbar-background: var(--ds-background-body);
   --toolbar-border-color: var(--ds-divider);
-  --toolbar-font-color: var(--ds-text-primary);
-  --toolbar-muted-color: var(--ds-text-disabled);
+  --toolbar-font-color: var(--ds-primary-main);
+  --toolbar-muted-color: var(--ds-text-tertiary);
```

Comment:
Nice change using the tokens! Lol I'm not sure why I set that to the disabled color